docs/readmes/lte/upgrade_1_6.md

```diff
@@ -0,0 +1,37 @@
+---
+id: upgrade_1_6
+title: Upgrade to v1.6
+hide_title: true
+---
+
+# Upgrade to v1.6
+
+### Fresh Install Notes
+
+For fresh gateway installs, please use the following install scripts.
+
+```bash
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_ubuntu.sh
+chmod +x agw_install_ubuntu.sh
+./agw_install_ubuntu.sh
+```
```

Comment:
Don't denormalize this description. Just provide a link to the relevant docusaurus page